src/PatientChecking/PatientChecking/Startup.cs

```diff
@@ -60,7 +60,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             app.UseSwaggerUI(c =>
             {
                 c.SwaggerEndpoint("/swagger/v1/swagger.json", "My API V1");
-                c.RoutePrefix = string.Empty;
+                //c.RoutePrefix = string.Empty;
             });
```

Comment:
Please remove all comments that are not intended to let reader to be aware of.

Reply:
comments have been removed